packages/instant-apps-components/src/components/instant-apps-header/instant-apps-header.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, test } from 'vitest';
+
+import '../../../dist/components/instant-apps-header.js';
+const header = document.createElement('instant-apps-header');
+
+test('should render component with custom title text', async () => {
+  const customTitle = 'Instant Apps Header';
+  header.titleText = customTitle;
+  document.body.appendChild(header);
+  await new Promise(resolve => requestIdleCallback(resolve));
+  
+  expect(header.titleText).toBe(customTitle);
+});
+
+test('test info button visibility', async () => {
+  header.infoButton = true;
+  header.infoTitleText = "info test"
+  document.body.appendChild(header);
+  await new Promise(resolve => requestIdleCallback(resolve));
+  header.toggleInfo();
+  expect(header.infoButton).toBe(true);
+  expect(header.infoIsOpen).toBe(true);
+  header.toggleInfo();
+  expect(header.infoIsOpen).toBe(false);
+  const infoElement = header.shadowRoot?.querySelector('#infoButton');
+  expect(infoElement? true : false).toBe(true);
```

Comment:
you can use `toBeTruthy()` or `toBeFalsy()` instead of testing for a boolean since that's what you're intending to check.